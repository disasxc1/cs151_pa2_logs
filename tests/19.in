2 1 1
3 2 2
4 3 3
777 7 8
22 1 0
33 2 1
44 3 2
55 4 3
99 5 4
98 6 4
97 7 4
96 8 4
95 9 4
11 7 5
105 6 8
106 6 9
888 4 4
107 5 9
12 7 6
13 7 7
14 8 7
15 9 7
16 10 7
100 13 2
104 15 2
101 14 2
102 14 3
103 14 1

