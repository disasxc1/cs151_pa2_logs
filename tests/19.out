2 3 4 11 12 13 14 15 16 22 33 44 55 95 96 97 98 99 105 106 107 777 888
100 101 102 103 104

